client-src/elements/chromedash-all-features-page_test.ts

```diff
@@ -5,28 +5,28 @@ import './chromedash-toast';
 import {ChromeStatusClient} from '../js-src/cs-client';
 import sinon from 'sinon';
 
+window.csClient = new ChromeStatusClient('fake_token', 1);
+let getStarsStub = sinon.stub(window.csClient, 'getStars');
```

Comment:
I'm surprised it works to create the stub once, restore it in the `afterEach`, and then expect to get the stubbed behavior in later tests.